atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/ConflictHandler.java

```diff
@@ -47,7 +47,7 @@
      * Note: This ConflictHandler has the ABA problem (https://en.wikipedia.org/wiki/ABA_problem).  The easiest
      * way to fix this is to have the value always increase.
      */
-    RETRY_ON_VALUE_CHANGED,
+    RETRY_ON_VALUE_CHANGED(false, true, true, false),
```

Comment:
Is this handler truly the same as `RETRY_ON_WRITE_WRITE`?

Reply:
Ah, I see the difference is in the exception handling [here](https://github.com/palantir/atlasdb/pull/3267/files#diff-bf062bd4c13b8c0022080ca8ef49c4d2R1515).